src/Akka.Persistence.MongoDb/Journal/MongoDbJournalQueries.cs

```diff
@@ -0,0 +1,225 @@
+﻿// -----------------------------------------------------------------------
+//   <copyright file="MongoDbJournalQueries.cs" company="Petabridge, LLC">
+//     Copyright (C) 2015-2023 .NET Petabridge, LLC
+//   </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using MongoDB.Bson;
+using MongoDB.Driver;
+using MongoDB.Driver.Linq;
+
+#nullable enable
+namespace Akka.Persistence.MongoDb.Journal;
+
+internal static class MongoDbJournalQueries
```

Comment:
Queries are refactored into this static class

Reply:
LGTM